### PR TITLE
rework two funcref code examples

### PR DESCRIPTION
**Pull request by matt08-prog**
Reworked two funcref code blocks as discussed in #6444. I don't have a lot of C# experience, however, so this only fixed the gdscript examples.

### CODE REVIEW

**Comment by Vennnot** [comment]
Can this be merged?

**Comment by mhilbrunner** [comment]
No, because I think its incorrect, when referencing a function in Godot 4, you use its name, but without parentheses as thats a function call then. Will see about fixing this.

File: tutorials/best_practices/godot_interfaces.rst [hunk 1]
```diff
@@ -464,15 +464,15 @@ accesses:
...
+            fn.call()
 
     # parent.gd
     extends Node
 
     @onready var child = $Child
 
     func _ready():
-        child.fn = funcref(self, "print_me")
+        child.fn = print_me()
```
Comment (mhilbrunner) [inline tutorials/best_practices/godot_interfaces.rst:475]:
```suggestion
        child.fn = print_me
```

**Comment by mhilbrunner** [comment]
Fixed and merged. Thanks for contributing!

**Pull request merged**